pkgs/development/python-modules/twisted/default.nix

```
@@ -48,11 +48,12 @@ buildPythonPackage rec {
   pname = "twisted";
   version = "22.10.0";
   format = "setuptools";
+  wheelPname = "Twisted";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
-    pname = "Twisted";
+    pname = wheelPname;
```

```suggestion
    pname = wheelName;
```
not sure if that is better or if we should rename the attr in fetchPypi

---

I think `wheelPname` is better because it is more intuitive that you need to put the value that you would be putting in `pname` and not in `name`. The wheel files that are generated include the pname and version (aka name), so here we really care about the pname part.

---

> not sure if that is better or if we should rename the attr in fetchPypi

Most times the value will be the same. I can add that attribute there, and assert you're setting only one of both. Since those derivations are FODs, changes there shouldn't produce mass rebuilds.